src/main/java/com/mongodb/kafka/connect/source/StartedMongoSourceTask.java

```diff
@@ -85,10 +89,10 @@
 import com.mongodb.kafka.connect.util.time.InnerOuterTimer.InnerTimer;
 
 final class StartedMongoSourceTask implements AutoCloseable {
-  private static final Logger LOGGER = LoggerFactory.getLogger(MongoSourceTask.class);
   private static final String FULL_DOCUMENT = "fullDocument";
   private static final int NAMESPACE_NOT_FOUND_ERROR = 26;
   private static final int ILLEGAL_OPERATION_ERROR = 20;
+  private static final int UNKNOWN_FIELD_ERROR = 40415;
```

Comment:
Is this the error for setting startAtOperationTime pre 4.0?

Reply:
Yes. But it may also happen with other fields we may introduce in the future, i.e., we should not make the error message specific to `startAtOperationTime`.